test/test_functional_tensor.py

```diff
@@ -69,29 +69,12 @@ class TestRotate:
     @pytest.mark.parametrize("device", cpu_and_gpu())
     @pytest.mark.parametrize("height, width", [(7, 33), (26, IMG_W), (32, IMG_W)])
     @pytest.mark.parametrize(
-        "center",
```

Comment:
Although I appreciate this cleanup, it is unrelated to the PR. Could you please revert this? We will happily accept a follow-up PR with the changes.